source/guides/ember-cli/installing.md

```diff
@@ -0,0 +1,62 @@
+## Got Node (and NPM)?
```

Comment:
Excellent feedback! :) I cleaned it up locally and agree it makes much more sense under the troubleshooting